README.md

```diff
@@ -30,7 +32,7 @@ INSTALLED_APPS = [
 
 ### BitGo
 
-To use the BitGo's wallet, it is necessary to import `BitGoIntegration` class from `polaris_bitgo.bitgo` package and pass it as an argument to the `custodial` parameter in the Polaris' `register_integrations` function.
+To use the BitGo's wallet, it is necessary to import `BitGoIntegration` class from `polaris_bitgo.bitgo.integration` package and pass it as an argument to the `custody` parameter in the Polaris' `register_integrations` function.
```

Comment:
I'm a bit confused with the meaning of the words `argument` and `parameter` here. They both mean the same to me, the value passed to a function, so I don't really understand how to pass an argument to a parameter in a function (but this could be just a misconception from my part, anyway). 🤔 
Is the idea that the `BitGoIntegration` class' object should be passed to `register_integrations()` through the `custody` parameter?

Reply:
Nice catch! And answering your question, yes, the `BitGoIntegration` class instance should be passed to `register_integration` function. I'll fix that description.